app/main.js

```diff
@@ -24,6 +24,14 @@ if (
   require('electron-debug')();
 }
 
+app.setAsDefaultProtocolClient('bob');
+
+// Protocol handler for osx
```

Comment:
Is this osx only for now?  Ping me on Signal and I can test Windows whenever it's ready.

Reply:
just added support for window as well